samples/cellular/lwm2m_client/fota_external_mcu.rst

```diff
@@ -0,0 +1,285 @@
+      |       COEX2 | -- nRF interface line 8 -- | P1.15          |
+      |       P0.24 | -- nRF interface line 9 -- | P0.18 (nRESET) | (in v0.14.0 or later)
+   */
+
+SMP and MCUmgr protocols are used to communicate over the UART.
+nRF9160 is configured as a MCUMgr client and nRF52840 is configured as a server.
+
+External MCU FOTA options
+=========================
+
+Advanced Firmware Update object supports two instances by default.
+To support extrenal DFU target, extra instance is defined as follows:
+
++-------------+-------------+---------------------------------------------------------------------+
+| Instance ID | Owner       | DFU types                                                           |
++=============+=============+=====================================================================+
++| 0          | Application | DFU_TARGET_IMAGE_TYPE_MCUBOOT                                       |
++-------------+-------------+---------------------------------------------------------------------+
++| 1          | Modem       | DFU_TARGET_IMAGE_TYPE_MODEM_DELTA, DFU_TARGET_IMAGE_TYPE_FULL_MODEM |
++-------------+-------------+---------------------------------------------------------------------+
++| 2          | SMP         | DFU_TARGET_SMP                                                      |
++-------------+-------------+---------------------------------------------------------------------+
+
+Sample support firmware update from nrf9160DK to nRF52840 MCU. External Fota SMP adds new instance to Advanced Fota object and registers that to Device management server.
+
+Configuration
+-------------
+
+* :kconfig:option:`CONFIG_APP_SMP_CLIENT_FOTA_EXTERNAL` - Enable external SMP FOTA client
+* :kconfig:option:`CONFIG_APP_SMP_SERVER_RECOVERY_MODE` - Enable Reset line to activate MCUBoot recovery.
+
+
+Sample supports nRF9160DK UART2 connection to onboard nRF52840 with or withour MCUBoot recovery mode.
+nRF9160 device needs to enable UART2 on the Devicetree and Recovery mode overlay files.
+
+* :file:`overlay-mcumgr_client.conf` Define configuration for External Fota client
+  This requires :file:`nrf9160dk_mcumgr_client_uart2.overlay` devicetree extra configuration.
+* :file:`overlay-mcumgr_reset.conf` Enable MCUBoot recovery mode
+  This requires :file:`nrf9160dk_recovery.overlay` devicetree extra configuration.
+
+
+For external Fota feature following overlay configurations needs to be activated as well:
+* :file:`overlay-adv-firmware.conf` - Enables experimental Advanced Firmware Update object.
+* :file:`overlay-lwm2m-1.1.conf` - Enables experimental Advanced Firmware Update object.
+
+Also one of the Coiote Device Management`_ server configuration is required:
+* :file:`overlay-avsystem.conf` - Uses `Coiote Device Management`_ server
+* :file:`overlay-avsystem-bootstrap.conf` - Uses Coiote in bootstrap mode.
+
+Build for MCUBoot recovery mode with Bootstrap
+----------------------------------------------
+
```

Comment:
You need an introductory text even if the heading says it all (for all of these sections). Something like:

"To build for MCUBoot recovery mode with bootstrap, use the following command:"

And I believe you do not need to capitalize bootstrap.